### PR TITLE
[musicbrainzcovers] Unbundle musicbrainzngs library

### LINKED ISSUE #203: [musicbrainzcovers plugin] Update or unbundle musicbrainzngs
Exaile's version of [musicbrainzngs](https://github.com/alastair/python-musicbrainzngs) hasn't seen an update in 3 years. When fetching covers with musicbrainzngs enabled I'm getting many of these messages:
```
12:42:26,610:DEBUG   : in <ws2:release>, uncaught <text-representation> (python-musicbrainz-ngs)
12:42:26,610:DEBUG   : in <ws2:release>, uncaught <artist-credit> (python-musicbrainz-ngs)
12:42:26,610:DEBUG   : in <ws2:release>, uncaught <release-group> (python-musicbrainz-ngs)
12:42:26,610:DEBUG   : in <ws2:release>, uncaught <release-event-list> (python-musicbrainz-ngs)
12:42:26,610:DEBUG   : in <ws2:release>, uncaught <label-info-list> (python-musicbrainz-ngs)
12:42:26,611:DEBUG   : in <ws2:release>, uncaught <medium-list> (python-musicbrainz-ngs)
12:42:26,611:DEBUG   : in <ws2:release>, not delegating <title> (python-musicbrainz-ngs)
12:42:26,611:DEBUG   : in <ws2:release>, not delegating <status> (python-musicbrainz-ngs)
12:42:26,611:DEBUG   : in <ws2:name-credit>, uncaught <artist> (python-musicbrainz-ngs)
12:42:26,612:DEBUG   : in <ws2:artist>, not delegating <name> (python-musicbrainz-ngs)
12:42:26,612:DEBUG   : in <ws2:artist>, not delegating <sort-name> (python-musicbrainz-ngs)
12:42:26,612:DEBUG   : in <ws2:release-group>, not delegating <primary-type> (python-musicbrainz-ngs)
```
Which looks like API changed a lot.

This plugin is available on several distributions:
* [pypi/pip](https://pypi.python.org/pypi/musicbrainzngs/0.5)
* [Debian](https://packages.debian.org/stable/python/python-musicbrainzngs)
  * [Ubuntu](https://launchpad.net/ubuntu/+source/musicbrainzngs)
* [Arch Linux](https://www.archlinux.org/packages/community/any/python2-musicbrainzngs/)
* [Fedora](https://apps.fedoraproject.org/packages/python-musicbrainzngs)

I suggest either updating or unbundling this library. I guess this change should happen before publishing exaile 4.0 since shipping a 3 years old musicbrainzngs is probably a bad idea. I'd prefer unbundling if that's not a major issue for Windows users.

This is not critical since afaik we only fetch covers from musicbrainzngs. I tested fetching covers with the new API (version 0.5 from Fedora) and it works fine.

What's your opinion on that?

### PR DESCRIPTION
**Pull request by genodeftest**
Exaile's version of musicbrainzngs is outdated, upstream has fixes for many warnings.
Rely on pip and Linux distributions to provide musicbrainzngs.

Fixes #203 

Tested: Fetching covers works fine with musicbrainzngs 0.5 (Fedora).

Please note:
* Changes in DEPS might conflict with my other pull request (#204)
* I'm not sure about the syntax in `tools/installer/requirements.txt`. Can you please verify that?

### CODE REVIEW

File: tools/installer/requirements.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 mutagen==1.31
+ensurepip
```
Comment (virtuald) [inline tools/installer/requirements.txt]: No. This should be the name of the module that's required. Probably should put pylast in here too, actually...

Comment (genodeftest) [inline tools/installer/requirements.txt]: Oh, ok. I thought this is correct since `ensurepip` *is* a module name.

Comment (genodeftest) [inline tools/installer/requirements.txt]: What about
* lxml required by lyricsmania
* BeautifulSoup required by lyricswiki
* grp, cairo required by daapserver
* cairo (again), required by some other plugins
* pyHook required by winmmkeys (only useful on Windows, obviously)

Ideally this list would be autogenerated to reduce maintenance and make errors less likely.

Comment (virtuald) [inline tools/installer/requirements.txt]: This list is for things that get installed into the SDK environment when building the windows and OSX frozen versions of Exaile, and is not used for any other purpose. The only things that can go here are python modules that can be installed via pip (eg, exist on pypi.python.org) on Windows and OSX. 

* ensurepip cannot be installed by pip, it is a builtin module
* lxml has binary dependencies, should not be on this list
* BeautifulSoup... probably cannot be installed
* cairo cannot be installed by pip, and should not be in this list
* pyHook... I don't know the answer to that
* Not sure what grp is. Does DAAP work?

I don't think its worth the time/effort to autogenerate this list, as it's not a particularly long list.

**Comment by genodeftest** [comment]
I think it is fixed now.

**Pull request merged**